app/views/admin/induction_periods/edit.html.erb

```diff
@@ -18,14 +18,11 @@
       <%= f.govuk_date_field :finished_on,
           legend: { text: "End date", size: "m" } %>
 
-      <%= f.govuk_number_field :number_of_terms,
+      <%= f.govuk_text_field :number_of_terms,
```

Comment:
I was able to type in letters without a validation catching it which resulted in `0.0` being saved. I think this introduces the risk of collecting bad data. 

`4.r5` becomes `4.0` so typos could result in subtle deviations. Am I right in thinking this could ultimately impact payments?

Reply:
Yeah - we should be using `input type="text"` here. There's [some guidance in the design system on the topic here](https://design-system.service.gov.uk/components/text-input/#asking-for-decimal-numbers).